src/connection.h

```diff
@@ -142,22 +120,20 @@ class Connection : public std::enable_shared_from_this<Connection>
 
 		std::recursive_mutex m_connectionLock;
 
+		std::queue<OutputMessage_ptr> messageQueue;
```

Comment:
I would prefer std::list here, because queue defaults to a deque as its underlying container and we don't need random access in the message queue.